lib/model.d.ts

```diff
@@ -2497,9 +2497,9 @@ export abstract class Model {
    * @param options.reset Clear all previously set data values
    */
   set<K extends keyof this>(key: K, value: this[K], options?: SetOptions): this;
-  set(keys: Partial<this>, options?: SetOptions): this;
+  set(keys: object, options?: SetOptions): this;
```

Comment:
I don't quite see the reason for this change, can `set` change attributes of included models? Afaik it can indeed only take attributes of its class